Makefile

```diff
@@ -244,22 +244,89 @@ verify-crd:
+	mv '$(3)'/scylla-operator/templates/edit_clusterrole.yaml '$(2)'/00_scyllacluster_clusterrole_edit.yaml
+	mv '$(3)'/scylla-operator/templates/scyllacluster_member_clusterrole.yaml '$(2)'/00_scyllacluster_member_clusterrole.yaml
+
+	mv '$(3)'/scylla-operator/templates/issuer.yaml '$(2)'/10_issuer.yaml
+	mv '$(3)'/scylla-operator/templates/certificate.yaml '$(2)'/10_certificate.yaml
+	mv '$(3)'/scylla-operator/templates/validatingwebhook.yaml '$(2)'/10_validatingwebhook.yaml
+	mv '$(3)'/scylla-operator/templates/service.yaml '$(2)'/10_service.yaml
+	mv '$(3)'/scylla-operator/templates/serviceaccount.yaml '$(2)'/10_serviceaccount.yaml
+	mv '$(3)'/scylla-operator/templates/pdb.yaml '$(2)'/10_pdb.yaml
+
+	mv '$(3)'/scylla-operator/templates/clusterrolebinding.yaml '$(2)'/20_clusterrolebinding.yaml
+
+	mv '$(3)'/scylla-operator/templates/deployment.yaml '$(2)'/50_deployment.yaml
+
+	@leftovers=$$( find '$(3)'/scylla-operator/ -mindepth 1 -type f ) && [[ "$${leftovers}" == "" ]] || \
+	( echo -e "Internal error: Unhandled helm files: \n$${leftovers}" && false )
+endef
+
+# $1 - Helm values file
+# $2 - output_dir
+# $3 - tmp_dir
+define generate-manager-manifests
+	$(call generate-manifests-from-helm,scylla-manager,helm/scylla-manager,$(1),$(3))
+
+	mv '$(3)'/scylla-manager/templates/controller_clusterrole.yaml '$(2)'/00_controller_clusterrole.yaml
+
+	mv '$(3)'/scylla-manager/templates/controller_serviceaccount.yaml '$(2)'/10_controller_serviceaccount.yaml
+	mv '$(3)'/scylla-manager/templates/controller_pdb.yaml '$(2)'/10_controller_pdb.yaml
+	mv '$(3)'/scylla-manager/templates/manager_service.yaml '$(2)'/10_manager_service.yaml
+	mv '$(3)'/scylla-manager/templates/manager_configmap.yaml '$(2)'/10_manager_configmap.yaml
+	mv '$(3)'/scylla-manager/charts/scylla/templates/serviceaccount.yaml '$(2)'/10_scyllacluster_serviceaccount.yaml
+
+	mv '$(3)'/scylla-manager/templates/controller_clusterrolebinding.yaml '$(2)'/20_controller_clusterrolebinding.yaml
+	mv '$(3)'/scylla-manager/charts/scylla/templates/rolebinding.yaml '$(2)'/20_scyllacluster_rolebinding.yaml
+
+	mv '$(3)'/scylla-manager/charts/scylla/templates/scyllacluster.yaml '$(2)'/50_scyllacluster.yaml
+	mv '$(3)'/scylla-manager/templates/controller_deployment.yaml '$(2)'/50_controller_deployment.yaml
+	mv '$(3)'/scylla-manager/templates/manager_deployment.yaml '$(2)'/50_manager_deployment.yaml
+
+	@leftovers=$$( find '$(3)'/scylla-manager/ -mindepth 1 -type f ) && [[ "$${leftovers}" == "" ]] || \
+	( echo -e "Internal error: Unhandled helm files: \n$${leftovers}" && false )
+endef
+
+update-deploy: tmp_dir:=$(shell mktemp -d)
+update-deploy:
+	$(call generate-operator-manifests,helm/deploy/operator.yaml,deploy/operator,$(tmp_dir))
+	$(call generate-manager-manifests,helm/deploy/manager_prod.yaml,deploy/manager/prod,$(tmp_dir))
+	$(call generate-manager-manifests,helm/deploy/manager_dev.yaml,deploy/manager/dev,$(tmp_dir))
```

Comment:
This won't verify e.g. the manual `00_namespace.yaml` file matches (or any future manual files) - would it be safer to cp `deploy/manager/prod` into  `deploy/manager/dev` and just `yq` edit the resources?

Reply:
also `helm/deploy/manager_prod.yaml` and `helm/deploy/manager_dev.yaml` could get out of sync